ci/bamboo/build-manylinux-prototype.sh

```diff
@@ -1,101 +0,0 @@
-#!/bin/bash
```

Comment:
Regarding
> I think the script makes more sense in the ci directory. You can leave it in ci/ if you don't want to put it in ci/bamboo/.

My initial thinking for placing it outside of CI is that I didn't want developers (inc 3rd party) to think that it's only for Continuous Integration systems. Developers should be able to discover and use this script to build the manylinux wheel without being mislead that it's for ci only. Also, by having it outside of `ci/`, we're less likely to change the script in such a way that it only works in our ci builds (ci-specific environment variables, etc.).